doc/general_marshaled_objects.md

```diff
@@ -87,6 +96,26 @@ The receiver of the above request can invoke `DoSomething` on the marshaled `ISo
 }
 ```
 
+In the above `DoSomething` example, we might suppose that `DoSomething` is always defined on this object.
+We might further suppose that because `subtypes: [1]` was specified, that some optional functionality is also exposed on that object.
+The functionality designated by `1` would be documented by the RPC contract owner and is outside the scope of this general protocol documentation.
+But supposing that `1` meant that the marshaled object also offered a `DoSomethingElse` method, the receiver of this object could then invoke:
+
+```json
+{
+    "jsonrpc": "2.0",
+    "id": 20,
+    "method": "$/invokeProxy/5/1.DoSomethingElse",
+    "params": []
+}
+```
+
+Note the `1.` prefix added to the leaf method name.
+This allows distinguishing groups of functionality on the object that may otherwise have colliding method names.
+Hosts of marshaled objects MUST support this prefix.
+Hosts MAY also support requests that omit the prefix, but should be prepared to handle any ambiguities that may arise.
```

Comment:
@AArnott, I think this is ok because you used "MAY", but I wanted to clarify that we are ONLY registering a `1.DoSomethingElse` RPC method on the server-side, not an unprefixed `DoSomethingElse`.

These are the registered RPC methods on the server-side for this example:
- `DoSomething`
- `1.DoSomething`
- `1.DoSomethingElse`

Reply:
Understood. This protocol doc may describe more (and less) than StreamJsonRpc itself directly supports. Another host (e.g. typescript) will very likely decide to support non-prefixed method names because they don't have interfaces at runtime and far more limited reflection than .NET has. I'm merely calling out this possibility.